packages/react/src/Grid/Grid.test.tsx

```diff
@@ -87,8 +88,23 @@ describe('Grid', () => {
     })
   })
 
+  gridTags.forEach((tag) => {
+    it(`renders with a custom ${tag} tag`, () => {
+      const { container } = render(<Grid as={tag} aria-label={tag === 'section' ? 'Accessible name' : undefined} />)
+
+      let component: HTMLElement | null
+      if (tag === 'div') {
+        component = container.querySelector(tag)
+      } else {
+        component = screen.queryByRole(ariaRoleForTag[tag])
```

Comment:
Why not use `getByRole` here?

Reply:
Wasn’t aware of the difference – updated.